object-group-game/Effect.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace object_group_game {
+	class Effect {
+		private string status;
+		private string displayName;
+		private int value;
```

Comment:
I think we should use C# Properties, https://stackoverflow.com/questions/4142867/what-is-the-difference-between-a-property-and-a-variable/4142957

This would remove getters and setters